### PR TITLE
Fixes broken links in skype.yaml

### PR DESCRIPTION
**Pull request by peteratticusberg**
also adds more skype emoticons

source: http://www.skaip.org/skype-emoticons

### CODE REVIEW

**Review by hugovk — CHANGES_REQUESTED** [review]
@peteratticusberg A lot of these images are too big to upload to Slack, and a few are badly named.

I suggest removing those which are too big, and renaming those with invalid names.

```
Found 128 errors in packs/skype.yaml
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ancientone.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/angel.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/anger.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/red.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/target.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/auld.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/bandit.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/beer.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/bhangra.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-$
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/bomb.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/bug.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/busyday.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/cake.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/unsee.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/cat.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/cheese.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/shivering.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/computer.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/golmaal.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: 8=
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: ;-
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/cwl.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: \:d/
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/dance.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/discodancer.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/disgust.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/doctorstrange.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/donkey.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/dream.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: |-
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ring.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: >:
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/facepalm.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/festiveparty.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/finger.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ghost.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/learn.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: =ǝe=
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/happy.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/hearteyes.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/hi.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/hug.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/hungrycat.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/idea.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ill.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/inlove.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/bollylove.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/island.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/wasntme.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/kaecilius.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/karlmordo.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/lfcfacepalm.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/lfclaugh.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/lfcparty.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ladyvampire.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/lang.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-d
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/laugh.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/listening.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/headphones.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/makeup.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/mariachilove.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/mmm.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/monkeygiggle.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/hungover.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/movember.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/muscle.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/muscleman.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-x
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/neil.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: b|
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ontheloo.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/party.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/pizza.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/polarbear.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/puke.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/pumpkin.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/reindeer.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/rofl.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sad.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sadness.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/santa.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sarcastic.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sheep.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/pig.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/chappal.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sloth.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/smoke.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: i=
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sleepy.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/snowangel.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-|
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/llsshock.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/heidy.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/stop.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/hero.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-o
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/suryannamaskar.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/swear.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/sweat.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/laddu.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/syne.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/tvbinge.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/ttm.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/taur.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/chai.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/mlt.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/tennisfail.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/time.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/tired.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/toivo.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: :-p
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/tongueout.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/unamused.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/vampire.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/whistle.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/whosthis.gif
Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: ;-
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/wonder.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/wong.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/worry.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/yawn.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/yotfr.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/yoga.gif
Error: must be smaller than 64K in file size: http://static.skaip.org/img/emoticons/v2/ffffff/zombie.gif
```
https://travis-ci.org/hugovk/emojipacks/builds/202392841#L713

**Comment by bergpete** [comment]
@hugovk very fair point, fired this off in haste 😢 . Fixed the issues per your comment.

File: packs/skype.yaml [hunk 1]
```diff
@@ -1,172 +1,358 @@
...
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/movie.gif
+- name: movinghome
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/movinghome.gif
+- name: '8'
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/music.gif
+- name: lipsealed
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/lipssealed.gif
+- name: ek
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/naturescall.gif
+- name: b|
```
Comment (hugovk) [inline packs/skype.yaml:177]: `|` is not allowed, custom emoji names can only contain lower case letters, numbers, dashes and underscores.

Comment (hugovk) [inline packs/skype.yaml]: `|` is not allowed, custom emoji names can only contain lower case letters, numbers, dashes and underscores.

**Review by hugovk — CHANGES_REQUESTED** [review]

**Comment by zachfeldman** [comment]
@peteratticusberg if you fix the above I'll probably merge this!

**Comment by zachfeldman** [comment]
Hey there @peteratticusberg , I'm closing this PR for now but feel free to open it back up later on if you'd like if it's still relevant!

**Pull request closed without merging**